src/components/organisms/ChatSidebar/components/PrivateChats/PrivateChats.tsx

```diff
@@ -6,15 +6,13 @@ import { AnyVenue } from "types/venues";
 
 import { WithId } from "utils/id";
 
-import {
-  usePrivateChatPreviews,
-  useOnlineUsersToDisplay,
-} from "hooks/privateChats";
-import { useChatSidebarControls } from "hooks/chatSidebar";
+import { useChatSidebarControls } from "hooks/chats/chatSidebar";
 
 import { PrivateChatPreview, RecipientChat, OnlineUser } from "..";
 
 import "./PrivateChats.scss";
+import { usePrivateChatPreviews } from "../../../../../hooks/chats/privateChats/usePrivateChatPreviews";
+import { useOnlineUsersToDisplay } from "../../../../../hooks/chats/privateChats/useOnlineUsersToDisplay";
```

Comment:
```suggestion
import { usePrivateChatPreviews } from "hooks/chats/privateChats/usePrivateChatPreviews";
import { useOnlineUsersToDisplay } from "hooks/chats/privateChats/useOnlineUsersToDisplay";
```

Reply:
And move them to other hooks

Reply:
It's just a grouping feature - they are all related to private chats so instead of putting them into general folder (which already contains a lot of hooks) I grouped them by purpose in a separate folder. The same is used in `hooks/auditorium` and `hooks/users`. 
Removed relative paths in https://github.com/sparkletown/sparkle/pull/1922/commits/518920cd8fb68353d12131d07f58a26bdf59ee3d

Reply:
@dmitrii-zolotukhin I mean move them to the top, to other hook imports

Reply:
moved in separate file in https://github.com/sparkletown/sparkle/pull/1922/commits/43b7b77a12a15f3a0fa0ccb7831da6e6648aa023